plugins/chat/spec/system/message_notifications_with_sidebar_spec.rb

```diff
@@ -12,12 +12,10 @@
   end
 
   def create_message(text: "this is fine", channel:, creator: Fabricate(:user))
-    using_session(creator.username) do
-      sign_in(creator)
-      chat_page.visit_channel(channel)
-      chat_channel_page.send_message(text)
-      expect(chat_channel_page).to have_message(text: text)
-    end
+    sign_in(creator)
```

Comment:
I wonder if this could lead to unexpected side-effects and change the logged user without us noticing. I see it's called inside the using_session block though.

Reply:
Yes thats the goal, it creates two browser tabs with different users logged in each tab

Reply:
I guess my concern is that `create_message` also calls `sign_in`. Someone may come in later and do something like:

```ruby
using_session(:user_1) do 
  create_message(channel: channel_1, creator: user_1)
  create_message(channel: channel_1, creator: user_2)
end
```

which could cause errors?

Reply:
yep I see what you mean, I will rethink a lot of page objects and helpers stuff later. About your concern, if you do this, you should get an instant failing test because the user won't be logged in and nothing will work.

Reply:
That's good enough then!

Reply:
thanks!